hummingbot/connector/exchange/wazirx/wazirx_exchange.py

```diff
@@ -0,0 +1,897 @@
+                if "ownTrade" in stream:
+                    trade_evt = event_message["data"]
+                    trade_msg = {
+                        "trade_id": trade_evt["t"],
+                        "order_id": trade_evt["o"],
+                        "traded_price": trade_evt["p"],
+                        "traded_quantity": trade_evt["q"],
+                        "fee_currency": trade_evt["U"],
+                        "fee": trade_evt["f"]
+                    }
+                    await self._process_trade_message(trade_msg)
+                elif "orderUpdate" in stream:
+                    order_evt = event_message["data"]
+                    order_msg = {
+                        "id": order_evt["i"],
+                        "status": order_evt["X"],
+                    }
+                    self._process_order_message(order_msg)
+                elif "outboundAccountPosition" in stream:
+                    balances = event_message["data"]["B"]
+                    for balance_entry in balances:
+                        asset_name = balance_entry["a"].upper()
+                        free_balance = Decimal(str(balance_entry["b"]))
+                        locked_balance = Decimal(str(balance_entry["l"]))
+                        self._account_balances[asset_name] = free_balance + locked_balance
+                        self._account_available_balances[asset_name] = free_balance
+            except asyncio.CancelledError:
+                raise
+            except Exception:
+                self.logger().error("Unexpected error in user stream listener loop.", exc_info=True)
+                await asyncio.sleep(5.0)
+
+    async def get_open_orders(self) -> List[OpenOrder]:
+        tracked_orders = {}
+        for order in list(self._in_flight_orders.values()):
+            tracked_orders[await order.get_exchange_order_id()] = order.client_order_id
+
+        result = await self._api_request(
+            "get",
+            CONSTANTS.GET_OPEN_ORDERS_PATH_URL,
+            {},
+            True
+        )
+        ret_val = []
+        for order in result["result"]:
+            if order["id"] in tracked_orders:
+                ret_val.append(
+                    OpenOrder(
+                        client_order_id=tracked_orders[order["id"]],
+                        trading_pair=wazirx_utils.convert_from_exchange_trading_pair(order["symbol"]),
+                        price=Decimal(str(order["price"])),
+                        amount=Decimal(str(order["origQty"])),
+                        executed_amount=Decimal(str(order["executedQty"])),
+                        status=order["status"],
+                        order_type=OrderType.LIMIT,
+                        is_buy=True if order["side"].lower() == "buy" else False,
+                        time=int(order["createdTime"]),
+                        exchange_order_id=order["id"]
+                    )
+                )
+        return ret_val

```

Comment:
I did not find any sender of this message. It might not be required.

Reply:
Thanks, removed.